dev_tool/src/dev.ts

```diff
@@ -196,6 +200,10 @@ function main() {
                                     type: 'boolean',
                                     describe: 'run postgres locally',
                                 })
+                                .option('otel', {
```

Comment:
This might also want it’s own stand-alone command ./dev local otel in addition to being an option for ./dev local. Not a requirement.